releng/server-lite-includes/pom.xml

```diff
@@ -0,0 +1,343 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.sap.xsk</groupId>
+		<artifactId>xsk-releng-parent</artifactId>
+		<version>0.17.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<name>XSK - Releng - Server</name>
+	<artifactId>xsk-server-lite-includes</artifactId>
```

Comment:
Shouldn't the changes in this PR be the default ones and change `releng/server` to `releng/server-full` or something similar? Also, shouldn't these changes be applied to all other `releng` modules?